Cython/Compiler/ParseTreeTransforms.py

```diff
@@ -889,6 +890,13 @@ def _check_valid_cython_module(self, pos, module_name):
             if module_name.startswith("cython." + wrong):
                 extra = "Did you mean 'cython.%s' ?" % correct
                 break
+        if not extra:
+            from .. import Shadow
+            if (submodule in Shadow.__dict__
+                    or submodule in Options.directive_types):
+                extra = ("'%s' is not a module but is accessible within the 'cython' module. "
+                    + "Did you mean 'from cython cimport %s' or just 'cimport cython'?") % (
```

Comment:
I found reading this error a bit overwhelming, too many words
Also probably no need to mention `from cython cimport floating` as the user wanted to use `cython.floating` and not `floating`
```suggestion
                extra = ("It's accessible in 'cython' without imports. Just make sure you 'cimport cython'") % (
```

Reply:
Thanks. I didn't use your exact wording but hopefully make it shorter and clearer.

Reply:
Seems good enough, I approve